http-clients/apache-client/src/main/java/software/amazon/awssdk/http/apache/ApacheHttpClient.java

```diff
@@ -133,6 +134,15 @@ public static Builder builder() {
         return new DefaultBuilder();
     }
 
+    /**
+     * Create a {@link HttpConnection} client with the default properties
+     *
+     * @return an {@link ApacheHttpClient}
+     */
+    public static SdkHttpClient create() {
+        return new DefaultBuilder().build();
+    }
```

Comment:
Can we just say ApacheHttpClient instead of HttpConnection? They shouldn't care about implementation details. Same with Netty below.